site/pages/search-results-fr.hbs

```diff
@@ -0,0 +1,240 @@
+			<ul>
+			<ol class="breadcrumb">
+				<li class="[link]">[domain name of search result]</li>
+				<li class="[link]">[parent breadcrumb of page]</li>
+				<!--Note: this always has 2 list items. Does not include other breadcrumb items.-->
+			<ol>
+			<p><time datetime="[dateModified of search result]">[DD MMM. YYYY]</time> - [description/highlight of search result. Search terms in <strong>]</p>
+		</section>
+		<section>
+			<h3><a href="[link]">[Search result page title (not including '- Canada.ca' or other institutional suffix)]</a></h3>
+			<ul>
+				<li class="label">[institution name]</li>
+				<li class="label">[institution name]</li>
+				<li class="label">[institution name]</li>
+				<!--Note: if labels go past 1 row, then option a) cut off at end of row, option b) do not show any labels beyond the first row. Text-overflow of the cut off label should be with an ellipsis. Expand/collapse to show all is not a requirement for Jan 2018 MVP-->
+				<!--Note 2: First label should always be the label of the context the search is in (if contextualised). Not necessarily order of tags in page metadata.-->
+			<ul>
+			<ol class="breadcrumb">
+				<li class="[link]">[domain name of search result]</li>
+				<li class="[link]">[parent breadcrumb of page]</li>
+				<!--Note: this always has 2 list items. Does not include other breadcrumb items.-->
+			<ol>
+			<p><time datetime="[dateModified of search result]">[DD MMM. YYYY]</time> - [description/highlight of search result. Search terms in <strong>]</p>
+		</section>
+		<section>
+			<h3><a href="[link]">[Search result page title (not including '- Canada.ca' or other institutional suffix)]</a></h3>
+			<ul>
+				<li class="label">[institution name]</li>
+				<li class="label">[institution name]</li>
+				<li class="label">[institution name]</li>
+				<!--Note: if labels go past 1 row, then option a) cut off at end of row, option b) do not show any labels beyond the first row. Text-overflow of the cut off label should be with an ellipsis. Expand/collapse to show all is not a requirement for Jan 2018 MVP-->
+				<!--Note 2: First label should always be the label of the context the search is in (if contextualised). Not necessarily order of tags in page metadata.-->
+			<ul>
+			<ol class="breadcrumb">
+				<li class="[link]">[domain name of search result]</li>
+				<li class="[link]">[parent breadcrumb of page]</li>
+				<!--Note: this always has 2 list items. Does not include other breadcrumb items.-->
+			<ol>
+			<p><time datetime="[dateModified of search result]">[DD MMM. YYYY]</time> - [description/highlight of search result. Search terms in <strong>]</p>
+		</section>
+		<section>
+			<h3><a href="[link]">[Search result page title (not including '- Canada.ca' or other institutional suffix)]</a></h3>
+			<ul>
+				<li class="label">[institution name]</li>
```

Comment:
It will be best to set the CSS class on the ```<ul>``` instead of each individual ```<li>```. Because that unordered list would only contains labels.